### DISCUSSION TITLE
Teslamate update newest version / waiting for postgres at database:5432 / not working anymore

### DISCUSSION THREAD

**Discussion by dansche** [teslamate-org/teslamate#3680] [Q&A]
Hi, just updated to newest release and have the above error, can anyone pointme to where what I should do?

Thanks

**Comment by dansche** [comment]
![docker compose file](https://github.com/teslamate-org/teslamate/assets/18419980/40f74f0e-5051-4f22-ba27-b3a12a8594e8)

**Comment by brianmay** [comment]
Can you post the full output when you try to start it?

**Reply by dansche** [reply]
Hey Brian, how embarassing, thanks for your offering to help, but your post made me look into all the parts running for Teslamate and I found out, that I run out of space on the SD-Card... so the database has not been able to start... after I deleted the last backup everything started up perfectly and runs again... Now I guess I need to clone the SD-Card to a bigger one.... Again thank you very much appreciating your pick up of the question.

**Reply by brianmay** [reply]
Glad you got it working :-)

**Comment by dansche** [comment]
closed as of now.... user failure....
